resources/views/admin/categories/index.blade.php

```diff
@@ -0,0 +1,101 @@
+@extends('layouts.app')
+
+@section('content')l
+    <!-- End Navbar -->
+    <div class="container-fluid py-4">
+        <div class="row">
+            <div class="col-12">
+                <div class="card mb-4">
+                    <div class="card-header pb-0">
+                        @if ($message = Session::get('success'))
+                          @if($message=="Category deleted successfully")                          
+                            <p class="alert alert-danger text-white" role="alert">{{ $message }}</p>     
+                          @else
+                            <p class="alert alert-success text-white" role="alert">{{ $message }}</p>                    
+                          @endif  
```

Comment:
Change these lines. kindly use it laravel session - >success message after redirection.